### DISCUSSION TITLE
Help to create InlineKeyboardMarkup

### DISCUSSION THREAD

**Discussion by DemaReaktor** [aiogram/aiogram#1330] [Q&A] [answered]
I just want to send message with InlineKeyboardMarkup, but always catch error. I tried to create as:
```
buttons = [[InlineKeyboardButton(text='Нічого')]]
        keyboard = InlineKeyboardMarkup(inline_keyboard=buttons)
```
and set: `bot.send_message(message.chat.id, "Що робити", reply_markup=Keyboards.main_keyboard())`
I imported classes as:
```
from aiogram.types.inline_keyboard_markup import InlineKeyboardMarkup
from aiogram.types.inline_keyboard_button import InlineKeyboardButton
```
But I catch error:
TypeError: Object of type InlineKeyboardMarkup is not JSON serializable

I tried use json and dumps to encode InlineKeyboardButton and It says it is not json serializable.
 So what do I do wrong and how to create InlineKeyboardMarkup without errors?

**Comment by DemaReaktor** [comment]
Ok. I tried use InlineKeyboardButton and InlineKeyboardMarkup from telebot and It works!!!!! 
But It is weird. Why create aiogram InlineKeyboard classes if I cant use them. Maybe I do it wrong, help me

**Reply by JrooTJunior** [reply, marked as answer]
Please provide more information about your environment.

Which version of the aiogram did you use?
Why did you mention telebot when you asked for help with the aiogram? 
Are you trying to use objects from one framework in another?
This is absolutely wrong, you cannot use aiogram objects in a telebot and vice versa. I don't see the `await' keyword in the first post, it looks like you are using a different library than aiogram.

**Reply by DemaReaktor** [reply]
Oh, I understood. I really use aiogram and telebot together. It should work now, thanks
